Sources/Plasma/PubUtilLib/plSurface/hsGMaterial.cpp

```diff
@@ -65,6 +67,9 @@ fLOD(0),
 fCompFlags(0),
 fLoadFlags(0),
 fLastUpdateTime(0)
+#if PLASMA_PIPELINE_GL || PLASMA_PIPELINE_METAL
```

Comment:
```suggestion
#if defined(PLASMA_PIPELINE_GL) || defined(PLASMA_PIPELINE_METAL)
```

Reply:
My suggestion (and I think what I did on the OpenGL branch) is to just unconditionally add the new field, and leave it set to null for DX

Reply:
This should no longer be conditional.